### PR TITLE
Added translation for levels 12 to 17 + tests

### PR DESCRIPTION
**Pull request by emil-meyknecht**
Push to finalize the translation structure up to level 17.
Issues #1453 , #1454 

Added levels 12 to 17 into the translation structure.

-Adds the translation for 'and', 'or' conditions.
-Adds the support for mathematical comparison signs.
-Adds the translation for the while loop
-Adds brackets around arrays after level 16
-Adds a semicolon after indentation in level 17

Tests available with test_translation_level_10.py to test_translation_level_17.py

### CODE REVIEW

File: hedy_translation.py [hunk 1]
```diff
@@ -28,8 +28,9 @@ def translate_keywords(input_string, from_lang="en", to_lang="nl", level=1):
     punctuation_symbols = ['!', '?', '.']
 
     keyword_dict = keywords_to_dict(to_lang)
-    if level > 7:
```
Comment (Felienne) [inline hedy_translation.py:31]: Why are we removing the level check here? processing blocks only needs to be done for levels over 7

**Review by Felienne — CHANGES_REQUESTED** [review]

File: hedy_translation.py (hunk 1 above)
Comment (emil-meyknecht) [reply hedy_translation.py:31]: I added the check that was already present in hedy.py, the preprocess_input() function. That one also only uses preprocess_blocks when level > 7

File: hedy_translation.py [hunk 2]
```diff
@@ -28,8 +28,9 @@ def translate_keywords(input_string, from_lang="en", to_lang="nl", level=1):
...
-        input_string = hedy.preprocess_blocks(input_string, level)
+
+    input_string = hedy.process_input_string(input_string, level)
```
Comment (emil-meyknecht) [inline hedy_translation.py:32]: This function

> I added the check that was already present in hedy.py, the preprocess_input() function. That one also only uses preprocess_blocks when level > 7

Comment (Felienne) [reply hedy_translation.py:32]: Ah of course!

**Review by Felienne — APPROVED** [review]

**Pull request merged**